app/queries/undo_submission_mutator.rb

```diff
@@ -5,12 +5,18 @@ class UndoSubmissionMutator < ApplicationQuery
 
   def undo_submission
     TimelineEvent.transaction do
+      owners = timeline_event.founders.load
       # Remove the submission
       timeline_event.destroy!
 
       # Set the most recent submission to latest.
-      last_submission = founder.timeline_events.where(target: target).order(created_at: :desc).first
-      last_submission.update!(latest: true) if last_submission.present?
+      owners.map do |owner|
```

Comment:
`.each`, not `.map`. I don't think you intend to return an array from this block.